frontend/src/features/admin-form/settings/components/PaymentSettingsSection/PaymentSettingsSection.tsx

```diff
@@ -11,18 +12,23 @@ import { FormResponseMode, PaymentChannel } from '~shared/types'
 
 import { BxsCheckCircle, BxsError, BxsInfoCircle } from '~assets/icons'
 import { GUIDE_PAYMENTS } from '~constants/links'
+import Checkbox from '~components/Checkbox'
 import FormLabel from '~components/FormControl/FormLabel'
 import InlineMessage from '~components/InlineMessage'
 import Input from '~components/Input'
 import Link from '~components/Link'
 
+import { useEnv } from '~features/env/queries'
+
 import { useAdminFormPayments, useAdminFormSettings } from '../../queries'
 
 import { BusinessInfoSection } from './BusinessInfoSection'
 import { StripeConnectButton } from './StripeConnectButton'
 
 const PaymentsAccountValidation = () => {
   const { hasPaymentCapabilities, isLoading, isError } = useAdminFormPayments()
+  const { data: { secretEnv } = {} } = useEnv()
```

Comment:
Hm true, since secret env is used by payments quite often

Reply:
I decided against this because we need secret env in places that use different hooks actually, so I just introduced a `const isProductionEnv = secretEnv === 'production'` variable so its cleaner here.